### PR TITLE
Added module Markdown documentation viewer

### PR DESCRIPTION
**Pull request by roadster31**
This PR add a module documentation viewer. The documentation is either the "standard" `Readme.md` or the content of the module's documentation directory, as declared by the `documentation` element in the `module.xml` file`.

Only Markdown (*.md) files are processed, other documentation formats are ignored.

If several files exists in the module's declared documentation directory, they're sorted in alphabetically ascending order before being concatenated and displayed to the user.

img-2015-03-13 12 02 14

### CODE REVIEW

File: core/lib/Thelia/Controller/Admin/ModuleController.php [hunk 1]
```diff
@@ -406,4 +407,64 @@ public function informationAction($module_id)
...
+            $finder = null;
+
+            // Check if the module has declared a documentation
+            $moduleDescriptor = $module->getAbsoluteConfigPath() . DS . 'module.xml';
+
+            if (false !== $xmlData = @simplexml_load_file($moduleDescriptor)) {
+                $documentationDirectory = (string) $xmlData->documentation;
+
+                if (! empty($documentationDirectory)) {
+                    $finder = Finder::create()->files()->in($module->getAbsoluteBaseDir())->name('*.md');
```
Comment (lovenunu) [inline core/lib/Thelia/Controller/Admin/ModuleController.php]: I think that here too, the name shouldn't be case sensitive, I'd really like to call my doc ```README.MD```.
I think ```"/.+\.md$/i"``` would be better

**Comment by lunika** [comment]
can you rebase your PR please ?

**Comment by roadster31** [comment]
Done.

**Comment by lunika** [comment]
Thanks

**Pull request merged**